src/controllers/question.js

```diff
@@ -1,11 +1,120 @@
+// const mongoose = require('mongoose');
+const Question = require('../models/question');
+const Student = require('../models/student');
+
+// function filterQuestions(query) {
+//   return async (_, res) => {
+//     try {
+//       // Find and sort by latest
+//       const questions = await Question.find(query); // .sort({ createdAt: -1 });
+//       if (questions) res.status(201).json(questions);
+//       else res.status(200).json({ messge: 'No questions found' });
+//     } catch (err) {
+//       res.status(422).json({ message: err.message });
+//     }
+//   };
+// }
+
+const filterQuestions = async (res, query) => {
+  try {
+    // Find and sort by latest
+    const questions = await Question.find(query); // .sort({ createdAt: -1 });
+    if (questions.length <= 0) {
+      res.status(200).json({ message: 'No questions found' });
+    } else res.status(200).json(questions);
+  } catch (err) {
+    res.status(422).json({ message: err.message });
```

Comment:
I think we should use another file for error messages or we can but then in the "env" file. hence; they will be the same messages for all of us. what do you think?

Reply:
yes, good idea